scipy/signal/ltisys.py

```diff
@@ -1024,37 +1490,168 @@ def to_tf(self, **kwargs):
 
         """
         return TransferFunction(*ss2tf(self._A, self._B, self._C, self._D,
-                                       **kwargs))
+                                       **kwargs), **self._dt_dict)
 
     def to_zpk(self, **kwargs):
         """
-        Convert system representation to `ZerosPolesGain`.
+        Convert system representation to `ZerosPolesGain`.
+
+        Parameters
+        ----------
+        kwargs : dict, optional
+            Additional keywords passed to `ss2zpk`
+
+        Returns
+        -------
+        sys : instance of `ZerosPolesGain`
+            Zeros, poles, gain representation of the current system
+
+        """
+        return ZerosPolesGain(*ss2zpk(self._A, self._B, self._C, self._D,
+                                      **kwargs), **self._dt_dict)
+
+    def to_ss(self):
+        """
+        Return a copy of the current `StateSpace` system.
+
+        Returns
+        -------
+        sys : instance of `StateSpace`
+            The current system (copy)
+
+        """
+        return copy.deepcopy(self)
```

Comment:
No need for a second method I think, that makes the API a bit ugly. 

> Because the lti class behaves the same way. 

OK, I see where this came from now. It's not a major problem, so let's just live with it.